web/templates/invite_user_modal.hbs

```diff
@@ -1,4 +1,9 @@
 <form id="invite-user-form">
+    <div class="setup-tips-container {{#unless (or is_admin is_owner)}}hide{{/unless}}">
+        <div class="setup-tips-org-desc"></div>
+        <div class="setup-tips-org-profile-pic"></div>
+        <div class="setup-tips-custom-profiles"></div>
```

Comment:
If we want to show only one banner at a time, can we just have a single div with a class name like "organization-setup-tips`?

Reply:
I think we want to show multiple banners, potentially (one banner for org description / profile pic + potentially a banner for custom profile fields).

Reply:
As mentioned by @alya we need to conditionally show the banners over here and found using different `divs` to be the simplest method of conditionally rendering and then removing the banners.